### ISSUE TITLE
Multiple formats within matrix

### ISSUE THREAD

**Issue by salwood82** [nazarovsky/mex-writematrix#1]
Does this function only support single formats within a matrix? I have a matrix with a mix of integers and floating point numbers

**Comment by nazarovsky** [comment]
Yes, the function supports only one format for all of the values in the
matrix.
